.history/query-engine/connector-test-kit-rs/query-tests-setup/src/runner/binary_20230116193406.rs

```diff
@@ -0,0 +1,274 @@
+use crate::{ConnectorTag, RunnerInterface, TestError, TestResult, TxResult};
+<<<<<<< HEAD
+=======
+use hyper::{Body, HeaderMap, Method, Request, Response};
+>>>>>>> 608455ced7 (Prepare the code for metrics capturing)
```

Comment:
as I saw this scrolling by: unhappy merge

Reply:
Good catch! This was me not pushing .gitignore for local vs history, all these files are a poor man's history extension of vs code.